server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -61,22 +61,26 @@ pub struct MessageIn {
     #[validate]
     pub event_type: EventTypeName,
     pub payload: serde_json::Value,
-
     #[validate(custom = "validate_channels_msg")]
     #[validate]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub channels: Option<EventChannelSet>,
+    #[validate(range(min = 5, max = 90))]
+    #[serde(rename = "payloadRetentionPeriod", default = "default_90")]
```

Comment:
No need to rename, it automatically camelCases everything (probably if not, but it should.
```suggestion
    #[serde(default = "default_90")]
```

Reply:
Double checked to make sure and it does automatically camelCase!